server/liveblog/blogs/blogs.py

```diff
@@ -32,7 +32,7 @@
 
 blogs_schema = {
     'title': metadata_schema['headline'],
-    'description': metadata_schema['description'],
+    'description': metadata_schema['description_text'],
```

Comment:
Our database field doesn't change. It is only pointing now to `description_text` which is defined in the superdesk schema